requirements/base.txt

```diff
@@ -17,10 +17,10 @@ python-novaclient>=7.1.2,<16
 tenacity>=6.2,<8
 
 # for hastexo_guacamole_client
-django<=4.2.8
+django<=4.2.14
 channels<=4.0.0
 daphne<=4.0.0
 twisted<24 # drop this restriction once we drop Python 3.8 and 3.9 support
```

Comment:
Should we perhaps use a `python_version` environment marker here too, as we do for `xblock-sdk` in `test.txt`?